pages/linux/rename.ul.md

```diff
@@ -0,0 +1,29 @@
+# rename.ul
+
+> Renames multiple files.
+> This is the util-linux version of `rename`. See `rename` for the perl version which is default on Debian.
```

Comment:
Do we really need to have it say "which is default on Debian"? I wouldn't say that's really necessary. What do you think?

Reply:
Removed it ;)